Shared/Sources/AutoResizingMaskViewSource.swift

```diff
@@ -118,11 +116,61 @@ final class AutoResizingMaskViewSource {
     return rightFlexibleTopBottomView
   }()
 
-  private func bakeViewWithFrame(_ frame: CGRect) -> UIView {
-    let view = UIView(frame: frame)
+  private func bakeViewWithFrame(_ frame: CGRect) -> UserInterfaceStyleAwareView {
+    let view = UserInterfaceStyleAwareView(frame: frame)
```

Comment:
I know you didn't change these particular lines and this is all over the codebase but avoid `type(of: self)` unless strictly necessary.

`static` properties can't be overridden in subclasses and thus the dynamic nature of the type is not necessary (only `class` properties can be overridden).  I also note that this class is final anyway so it makes it doubly icky.

Also, just because something is a "constant" doesn't mean it needs to be a class/static property.  When modeling the type think about whether these properties are properties of the "type" or of instances of the type.  It could be argued that these are constant properties of the instance.  Sure they are shared across all instances but is that just because they are constant and preinitialized?